shared/teams/team/container.js

```diff
@@ -63,6 +65,12 @@ const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, setRouteState, rout
   _loadTeam: teamname => dispatch(Creators.getDetails(teamname)),
   _onAddPeople: (teamname: Constants.Teamname) =>
     dispatch(navigateAppend([{props: {teamname}, selected: 'addPeople'}])),
+  _onAddSelf: (teamname: Constants.Teamname, you: ?string) => {
+    if (you) {
+      dispatch(navigateAppend([{props: {teamname}, selected: 'addPeople'}]))
+      dispatch(Search.addResultsToUserInput('addToTeamSearch', [you]))
```

Comment:
not sure if this logic belongs here, or if I should hand off `you` to the `addPeople` screen and let it add add it to the search results.